librocksdb-sys/rocksdb_lib_sources.txt

```diff
@@ -153,6 +153,12 @@ options/options.cc
 options/options_helper.cc
 options/options_parser.cc
 port/port_posix.cc
+port/win/env_default.cc
+port/win/env_win.cc
+port/win/io_win.cc
+port/win/port_win.cc
+port/win/win_logger.cc
+port/win/win_thread.cc
```

Comment:
Files related to Windows are added by build.rs script on windows host.

Reply:
Removed them